flocker/node/test/test_deploy.py

```diff
@@ -619,7 +619,9 @@ def test_does_not_exist(self):
     name=APP_NAME2,
     image=DockerImage.from_string(UNIT_FOR_APP2.container_image)
 )
-EMPTY_NODESTATE = NodeState(hostname=u"example.com")
+EMPTY_NODESTATE = NodeState(hostname=u"example.com",
+                            manifestations={}, applications=[],
+                            used_ports=[], devices={}, paths={})
```

Comment:
Also - pass a uuid here.  And hey this looks kinda like `EMPTY_NODE_STATE` so you could just move it to `flocker.node.testtools` and use it from both here and in `test_no_devices_one_dataset` above.